core/server/src/main/java/alluxio/worker/block/BlockWorkerClientRestServiceHandler.java

```diff
@@ -63,14 +66,17 @@
   public static final String UNLOCK_BLOCK = "unlock_block";
   public static final String WRITE_BLOCK = "write_block";
 
-  private final BlockWorker mBlockWorker = AlluxioWorkerService.Factory.get().getBlockWorker();
-  private final StorageTierAssoc mStorageTierAssoc = new WorkerStorageTierAssoc();
+  private final BlockWorker mBlockWorker;
+  private final StorageTierAssoc mStorageTierAssoc;
 
   /**
-   * Constructs a new {@link BlockWorkerClientRestServiceHandler}.
+   * @param context context for the servlet
```

Comment:
The original documentation was boilerplate to satisfy our javadoc requirement, not providing any value